src/crossfilter.js

```diff
@@ -121,6 +138,11 @@ function crossfilter() {
     offset = tmp.offset;
     one = tmp.one;
     zero = ~one;
+      
+    // Expose the dimension mask for internal use
+    dimension._offset = offset;
```

Comment:
Good point. Either way I'm good
On Fri, Mar 18, 2016 at 3:18 PM Ethan Jewett <notifications@github.com>
wrote:

> In src/crossfilter.js
> <https://github.com/crossfilter/crossfilter/pull/32#discussion_r56716138>:
>
> > @@ -121,6 +138,11 @@ function crossfilter() {
> >      offset = tmp.offset;
> >      one = tmp.one;
> >      zero = ~one;
> > +
> > +    // Expose the dimension mask for internal use
> > +    dimension._offset = offset;
>
> @tannerlinsley <https://github.com/tannerlinsley> That means it will be
> accessible on the external dimension API. I'd really prefer to keep the
> external API minimal. I think the original CF team did a really good job of
> that.
>
> @drarmstr <https://github.com/drarmstr> You're right - I had a brain
> malfunction :-) You'd need to maintain an array of dimension object
> references, and corresponding arrays of dimension offsets, etc. Then do an
> indexOf lookup on the dimension array and grab the corresponding index from
> the offset array. You'd want to maintain the arrays (splice) when
> dimension.dispose is called. If you want, I can do this after accepting the
> pull request onto a branch. Just let me know.
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/crossfilter/crossfilter/pull/32/files/9884cc984fa733f5b9760f57317a5aba7369280e#r56716138>
>